--- apachectl.orig	2015-02-23 12:34:21.458960242 +0100
+++ apachectl	2015-02-23 12:34:42.426959505 +0100
@@ -1,4 +1,12 @@
 #!/bin/sh
+### BEGIN INIT INFO
+# Provides:          apachectl
+# Required-Start:    $local_fs 
+# Required-Stop:     $local_fs
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: apachectl
+### END INIT INFO
 #
 # Licensed to the Apache Software Foundation (ASF) under one or more
 # contributor license agreements.  See the NOTICE file distributed with
