--- apachectl.orig      2015-03-27 17:50:10.086771875 +0100
+++ apachectl   2015-03-27 17:50:30.810771324 +0100
@@ -1,5 +1,10 @@
 #!/bin/sh
-#
+# chkconfig: 2345 20 80
+# description: apache daemon
+
+# Source function library.
+. /etc/init.d/functions
+
 # Licensed to the Apache Software Foundation (ASF) under one or more
 # contributor license agreements.  See the NOTICE file distributed with
 # this work for additional information regarding copyright ownership.
